verification/api/handler.go

```diff
@@ -472,3 +475,124 @@ func sendChallengeResponseSessionCreated(c *gin.Context, id string, jsonSession
 	c.Header("Location", path.Join("session", id))
 	sendChallengeResponseSessionWithStatus(c, http.StatusCreated, jsonSession)
 }
+
+func (o *Handler) getKey() (jwk.Key, error) {
+	var key jwk.Key
+
+	protoKey, err := o.Verifier.GetPublicKey()
+	if err != nil {
+		return key, err
+	}
+
+	key, err = jwk.ParseKey([]byte(protoKey.Key))
+	if err != nil {
+		return key, err
+	}
+
+	return key, nil
+
+}
+
+func (o *Handler) getVerificationMediaTypes() ([]string, error) {
+	mediaTypes, err := o.Verifier.SupportedMediaTypes()
+	if err != nil {
+		return mediaTypes, err
+	}
+
+	return mediaTypes, nil
```

Comment:
unless the `err != nil` branch is used to enrich the error context (e.g., `return mediaTypes, fmt.Errorf("SOME CONTEXT: %w", err)`) then this can be simplified to:
```suggestion
	return o.Verifier.SupportedMediaTypes()
```